src/prefect/utilities/schema_tools/hydration.py

```diff
@@ -1,7 +1,13 @@
 import json
 from typing import Any, Callable, Dict, Optional
 
-from pydantic import BaseModel, Field
+from prefect._internal.pydantic import HAS_PYDANTIC_V2
+
+if HAS_PYDANTIC_V2:
```

Comment:
Missed this in my earlier pr. thank you @zzstoatzz 🙇

Reply:
:blob_salute: